phone_home.c

```diff
@@ -263,7 +263,7 @@ Release * release_get_latest(void)
 	char *tag_name;
```

Comment:
Looking at the API documentation for json_string_value() tag_name should also be made const here.

Reply:
seems that a compiler check also agrees here about the `const`

Reply:
where and how? as far as I can see tag_name is only used for strcmp()

Reply:
This build: https://travis-ci.org/commodo/iio-oscilloscope/jobs/420955832
Apparently I stopped it before it finished ; I changed the `ent = readdir()` parentheses, then added the `const` qualifier (based on your comment), then stopped that build (to let a new one run).

Reply:
ok, sorry, misunderstood. Thought you meant the compiler did not agree.